doc/install.docker.markdown

```diff
@@ -18,3 +20,16 @@ Clone this repository, swith on good branch and run
       -e DISPLAY=$DISPLAY -e XAUTHORITY=$HOME/.Xauthority -e HOME=$HOME -e USER=$USER \
       -v $HOME:$HOME -v /tmp/.X11-unix:/tmp/.X11-unix -v /etc/sane.d/:/etc/sane.d/ \
       paperwork    
+
+# Troubleshooting
+
+## Docker Installation
+
+At least on some systems the packages `apparmor` and `cgroup-lite` are required.
+
+## Docker Image Execution
+
+    Unable to init server: Could not connect: Connection refused
+    (paperwork:22): Gtk-WARNING **: cannot open display: :0.0
+
+Run this command: `xhost local:root` (source: http://stackoverflow.com/questions/28392949/running-chromium-inside-docker-gtk-cannot-open-display-0)
```

Comment:
Ok for this time, but in the future, please try to avoid external links. External links tend to die. If there is something in this page worth mentionning, then it should be in this document directly.